src/authors.rs

```diff
@@ -6,7 +6,7 @@ use std::env;
 /// Taken from cargo and thus (c) 2018 Cargo Developers
 ///
 /// cf. https://github.com/rust-lang/cargo/blob/d33c65cbd9d6f7ba1e18b2cdb85fea5a09973d3b/src/cargo/ops/cargo_new.rs#L595-L645
-pub fn get_authors() -> Result<String, failure::Error> {
+pub fn get_authors() -> Result<(String, String), failure::Error> {
```

Comment:
i think we'd want to return a Map of some type so that it's more discoverable as to what these two strings are

Reply:
additionally, this edits code that we've taken directly from Cargo. for maintenance purposes it might be nicer to have the username logic live outside of this and treat this code a bit like a vendored dependency.